Dockerfile

```diff
@@ -1,24 +1,15 @@
-FROM balena/open-balena-base:v11.0.2
+FROM balena/open-balena-base:no-systemd-master
 
 EXPOSE 80
 
-RUN apt-get update \
-	&& apt-get install \
-		musl \
-		redis-server \
```

Comment:
You've removed the potential for a local redis server cache altogether with this PR - have you checked if that'll work within openBalena?

Reply:
@dfunckt was saying we're not using redis inside the container. but looking at https://github.com/balena-io/open-balena/blob/master/compose/services.yml#L80 it seems we do...

Reply:
`redis` can be removed and open-balena will be updated to point to the shared instance.